packages/integrations/useAxios/index.test.ts

```diff
@@ -419,4 +419,77 @@ describe.skipIf(isBelowNode18)('useAxios', () => {
 
     expectTypeOf(d.data.value).toEqualTypeOf<number | undefined>()
   })
+
+  describe('reactivity', () => {
+    it('url should be reactive', async () => {
+      const counter = ref(1)
+      const myUrl = computed(() => `https://jsonplaceholder.typicode.com/todos/${counter.value}`)
+      const { data, execute } = await useAxios(myUrl)
+      expect(data.value.id).toBe(1)
+      counter.value = 2
+      await execute()
+      expect(data.value.id).toBe(2)
+    })
+
+    it('refetch should be reactive when combined with url', async () => {
+      const counter = ref(1)
+      const myUrl = computed(() => `https://jsonplaceholder.typicode.com/todos/${counter.value}`)
```

Comment:
maybe we should not rely on jsonplaceholder here. 

`useFetch` tests use local http worker.

Reply:
I'm not quite sure how to implement this suggestion.

Reply:
Just import this mockServer like this

https://github.com/vueuse/vueuse/blob/1e153e936a6055d633becd3a685603b4492d8791/packages/core/useFetch/index.test.ts#L7

Mockserver implementation is here: https://github.com/vueuse/vueuse/blob/main/packages/.test/mockServer.ts

Server mocks requests to `example.com`. Example from `useFetch`: 
https://github.com/vueuse/vueuse/blob/1e153e936a6055d633becd3a685603b4492d8791/packages/core/useFetch/index.test.ts#L27-L34

If you are still unsure or need help feel free to ping me :)

Reply:
I believe I implemented it as suggested